packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -79,7 +79,7 @@ const TradersHub = observer(() => {
                     'traders-hub--mobile--eu-user': is_eu_user,
                 })}
                 height_offset='50px'
-                is_disabled={isDesktop()}
+                is_disabled={isDesktop}
```

Comment:
🤔

Reply:
I was reverting my changes after I figure out an alternative. I probably missed that 😂 . I'll amend it on the next commit 😃 👍🏼